### PR TITLE
Add feature rules are displayed when user clicks the first page - wzd - 4648 

### PR DESCRIPTION
**Pull request by mauceballos**
### Description
Added a Feature to increase coverage for the Rule module. This feature checks the paginator on the rules page.
 

### Evidence

<details>
<summary>test execution</summary>

</details>

### Check List
- [x] All tests pass
- [x] New functionality includes testing.
- [x] New functionality has been documented.

### CODE REVIEW

File: test/cypress/cypress/integration/features/management/rules/draft--check-rows-dropdow.feature [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Feature: Validate row dropdown on Rule page
+
+    As a wazuh user
+    i want to see the Rules pages
```
Comment (AlexRuiz7) [inline test/cypress/cypress/integration/features/management/rules/draft--check-rows-dropdow.feature:4]: In English, the first-person pronoun is always capitalized (I).

Comment (mauceballos) [reply test/cypress/cypress/integration/features/management/rules/draft--check-rows-dropdow.feature:4]: Done

**Review by AlexRuiz7 — APPROVED** [review]
Solved a few of grammatical errors.

LGTM!

**Comment by AlexRuiz7** [comment]
The file `test/cypress/cypress/integration/features/management/rules/draft--check-rows-dropdow.feature` was added to this pull request by mistake and must be removed.

**Comment by mauceballos** [comment]
Some Features are failing because misspellings on step features

solved #4682

**Pull request merged**